### PR TITLE
Update button

### LINKED ISSUE #464: Request: An update button directly visible instead of being in sub-menu
**Is your feature request related to a problem? Please describe.**
Even though HL clearly displays games that need updates, there is no way from that view to actually update the game(s) with one click. You need to go in the game details and the side menu to update the game. 

**Describe the solution you'd like**
A "Update game" button next to "Play Now" and "Uninstall" in the game details screen, and also a mini-icon in the overall game list to do the same thing.

### PR DESCRIPTION
**Pull request by dragonDScript**
I've added an update button and removed the update button on the three dot menu.

Fixes #464 

I couldn't check if it works correctly, because any of my games require to be updated. Before merging, it is important to check if it works. 
----------------------------------

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Created / Updated Tests
- [ ] Created / Updated documentation

Tested on a Clean Install with AppImage:
- [ ] Login
- [ ] Download a Game
- [ ] Import a Game
- [ ] Launch Default config
- [ ] Launch Personalized Config
- [x] Launch wine
- [ ] Launch Proton
- [ ] Launch Custom Proton
- [ ] Launch Tools (Winetricks and Winecfg)
- [ ] Uninstall Game
- [ ] Move Game
- [ ] Languages (Test if no translation stopped working)

### CODE REVIEW

**Comment by flavioislima** [comment]
I have a few games to update. I'll check if it works.
But would be needed to discuss the design of it first. Like where the button would be, color, etc.

**Comment by dragonDScript** [comment]
> I have a few games to update. I'll check if it works.
> But would be needed to discuss the design of it first. Like where the button would be, color, etc.

Of course! right now it is a normal button.

**Comment by flavioislima** [comment]
@dragonDScript finished on #480 since I could not send changes to your branch

File: src/screens/Game/GamePage/index.tsx [hunk 1]
```diff
@@ -305,6 +320,17 @@ export default function GamePage(): JSX.Element | null {
                         </button>
                       </>
                     )}
+                    {hasUpdate && is_installed && is_game && (
```
[the remark this answers is not in the document]
Comment (dragonDScript) [reply src/screens/Game/GamePage/index.tsx:323]: No. A button is a thing you can click and text is text. We shouldn't mix it. We can have both if you want. But a button is a clear signal of a clickable action.

**Pull request closed without merging**